contracts/FastBreak.cdc

```diff
@@ -0,0 +1,682 @@
+/*
+    Fast Break Game Contract
+    Author: Jeremy Ahrens jer.ahrens@dapperlabs.com
+*/
+
+import NonFungibleToken from 0xNFTADDRESS
+import TopShot from 0xTOPSHOTADDRESS
+
+/// Game & Oracle Contract for Fast Break
```

Comment:
Valiant effort on the comments, but what I meant when I asked for comments was to have comments that actually give some context on what things are for. like for each event have a description of the specific context when the event is emitted, or for a type definition, what the type is for and how it is used in practice, and comments for the unclear fields

Additionally, some comments at the top of the file defining some common terminology, like "run", "game", "fatigue", etc

Reply:
Absolutely, added depth to comments, especially field and game definitions.